components/mediation/tasks/org.wso2.micro.integrator.ntask.core/src/main/java/org/wso2/micro/integrator/ntask/core/internal/TasksDSComponent.java

```diff
@@ -88,6 +97,22 @@ protected void activate(ComponentContext ctx) {
             bundleContext.registerService(TaskService.class.getName(), getTaskService(), null);
             //            bundleContext.registerService(Axis2ConfigurationContextObserver.class.getName(), new TaskAxis2ConfigurationContextObserver(getTaskService()), null);
 
+            CarbonDataSource dataSource = dataSourceService.getDataSource("WSO2_COORDINATION_DB");
```

Comment:
Shall we add a sample datasource config here

Reply:
I have added sample TOML config in the PR description

Reply:
Shall we add the default data sources to toml , else adding this only will remove the rest